homeassistant/components/bt_smarthub/device_tracker.py

```diff
@@ -35,8 +39,12 @@ def __init__(self, config):
         """Initialise the scanner."""
         _LOGGER.debug("Initialising BT Smart Hub")
         self.host = config[CONF_HOST]
+        self.smarthub_model = config.get(CONF_SMARTHUB_MODEL)
```

Comment:
This isn't needed here. It's just needed when creating the `BTSmartHub` instance.

Reply:
Please see my latest commit which addresses this request.